### ISSUE TITLE
Package stats haven't updated in 24 hours

### ISSUE THREAD

**Issue by rianjs** [NuGet/NuGetGallery#6051]
The number has changed, but not by enough.

| Date | Download count | Change | 7d moving average|
|------------------|--------|-----|-----|
Fri, 8-Jun-2018 | 227,420 | 1,096 | 757 |
Sat, 9-Jun-2018 | 228,197 | 777 | 772 |
Sun, 10-Jun-2018 | 228,430 | 233 | 786 |
Mon, 11-Jun-2018 | 228,464 | 34 | 759 |
Tue, 12-Jun-2018 | 228,480 | 16 | 610 |

Weekends are lower, but not usually quite that low. I would guess whatever is updating the data has been choking since ~Sunday. Tuesday number is way, way too low.

(FWIW, the stats runs have been working well since #5089 in late December until now.)

Related issues from the past:

- https://github.com/NuGet/NuGetGallery/issues/5089
- https://github.com/NuGet/NuGetGallery/issues/4389
- https://github.com/NuGet/NuGetGallery/issues/4572

**Comment by skofman1** [comment]
@loic-sharma , can you please investigate?

**Comment by loic-sharma** [comment]
We've had lots of stats processing failures this weekend, which are being reprocessed now. I'll keep an eye on the reprocessing to make sure it goes smoothly.

**Comment by loic-sharma** [comment]
All statistics blobs have been reprocessed and Gallery downloads should be fixed after the next import. @rianjs Could you verify that your downloads are as expected tomorrow?

 I've opened the following issues to help mitigate this kind of problem in the future:

* https://github.com/NuGet/NuGetGallery/issues/6053
* https://github.com/NuGet/NuGetGallery/issues/6054

**Comment by rianjs** [comment]
The number has gone up, but not by enough, especially given that stats have been broken for a few days.

**Comment by rianjs** [comment]
No change at all in the numbers today, so stuff is still broken.

**Comment by rianjs** [comment]
Now we're going backwards:
- 228,927 yesterday
- 228,832 today.

**Comment by scottbommarito** [comment]
Hey @rianjs,

Sorry for the inconvenience.

I'm not sure why your download count went down--there's clearly some issue there. I checked our backend and it should have 228936 downloads right now, and not 228832. I have filed https://github.com/NuGet/NuGetGallery/issues/6069 to investigate this. I believe we've noticed this before but have never had the time to look into it.

We looked into the statistics pipeline more and found that there's been a massive drop in downloads being processed over the past couple days, despite all statistics failures from this weekend being reprocessed. We are investigating this now.
